storage/src/mmr/mem.rs

```diff
@@ -4,6 +4,8 @@
 use crate::mmr::hasher::Hasher;
 use crate::mmr::iterator::{nodes_needing_parents, PathIterator, PeakIterator};
 use crate::mmr::verification::Proof;
+use crate::mmr::Error;
+use crate::mmr::Error::{ElementPruned, InvalidElementPosition};
 use commonware_cryptography::{Digest, Hasher as CHasher};
 
 /// Implementation of `Mmr`.
```

Comment:
I think it might be worth leaving a note for the time being that running `Mmr` on 32-bit architectures will restrict the number of items stored to `u32` (as you'll overrun the vec).

We could do some fun stuff to get around this but think we should hold off because we'll implement this with storage-backing shortly.